bitnami/logstash/templates/ingress.yaml

```diff
@@ -1,5 +1,9 @@
 {{- if .Values.ingress.enabled -}}
+{{- if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
+apiVersion: networking.k8s.io/v1beta1
+{{- else -}}
 apiVersion: extensions/v1beta1
+{{- end }}
```

Comment:
We have a macro (`common.capabilities.ingress.apiVersion`) in the common library for this exactly purpose: https://github.com/bitnami/charts/blob/master/bitnami/common/templates/_capabilities.tpl#L16

However, it seems we're not using the common library as dependency on this chart (yet), so it's fine to me to use your approach at this moment.